### PR TITLE
Backport of fix for "[v10] t-call with template name in variable broken"

### PR DESCRIPTION
**Pull request by simahawk**
This is the fix for https://github.com/odoo/odoo/issues/16641. 
The fix is one line change, is tested, I tested it and is working. They are not willing to merge it in v10 tho.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      - wkhtmltopdf # report_webkit
+
+language: python
+sudo: false
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="9.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
```
Comment (pedrobaeza) [inline .travis.yml]: 10.0

**Comment by simahawk** [comment]
@moylop260 I added .travis.yml https://github.com/OCA/OCB/pull/654/commits/a2ede263ec003a2547aca6e8a69789c0fc396cbc
let's see if it works :)

**Comment by pedrobaeza** [comment]
I thought it was the travis timeout issue. Let's hope it doesn't happen in this branch.

**Comment by simahawk** [comment]
@pedrobaeza Moises told me to add it... let's see. If it makes travis happy we can port it to v9.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+install:
+  - ln -s ${HOME}/build/${ODOO_REPO} ${HOME}/${ODOO_REPO#*/}-${VERSION}
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - pip install -Ur ${HOME}/maintainer-quality-tools/requirements.txt
+  - pip install -Ur ${HOME}/build/${ODOO_REPO}/requirements.txt
+  - pip install -q QUnitSuite flake8 coveralls pylint
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - export INCLUDE=$(getaddons.py -m ${HOME}/build/${ODOO_REPO}/openerp/addons ${HOME}/build/${ODOO_REPO}/addons)
+  - export EXCLUDE=hw_scanner,hw_escpos
```
Comment (yajo) [inline .travis.yml]: I think [you need to exclude also `theme_bootswatch`](https://travis-ci.org/OCA/OCB/builds/282180916#L1142), right @moylop260?

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
now "2017-10-02 13:28:42,751 5417 ERROR openerp_test odoo.addons.base.ir.ir_qweb.assetsbundle: Could not execute command 'lessc'" 
@moylop260 aren't we installing it in travis setup?

anyway is getting next to green... hopefully :smile:

**Comment by moylop260** [comment]
I just installed npm less package.
let's see if there is another change required.

**Review by lepistone — APPROVED** [review]

**Comment by moylop260** [comment]
Could you squash the .travis.yml commits in one, please?

**Comment by simahawk** [comment]
@moylop260 thanks! Rebase done ;)

**Comment by yajo** [comment]
There are timeout errors on website's `test_20_crawl_demo` and `test_30_crawl_admin`. Is it possible that tests are not running with `--workers 0`?

**Pull request merged**